### ISSUE TITLE
Problems installing activity browser

### ISSUE THREAD

**Issue by moorotter** [LCA-ActivityBrowser/activity-browser#701]
Hi,

I am an absolute beginner in Python as well as in github so please excuse obvious mistakes I made.
I installed miniconda and use python 3.10.2, working on OS X.
I tried to install the activity browser, however, when I try to open it I get the following error:

File "/Users/****/opt/miniconda3/envs/ab_dev/lib/python3.10/site-packages/bw2data/project.py", line 58, in <module>
    class ProjectManager(collections.Iterable):
AttributeError: module 'collections' has no attribute 'Iterable'

During the installation, I read that I am supposed to use Pyhton 3.9 so I downgraded my python version from 3.10 to 3.9. When I try to install the activity-browser, it changes my version back to 3.10.

I don't really know how to get the activity browser running, any help would be greatly appreciated!
Thank you!

**Comment by marc-vdm** [comment]
Hi moorotter,

No worries! I'm not sure what you mean with "...it changes my version back to 3.10". I'll offer two options here, try the top one first, of that doesn't work, I hope the second one will.

1.
When using the `conda` terminal (black screen with white letters), just activate the environment with `conda activate ab_dev`. Then type: `conda install python=3.9`. This should hopefully put the right version back. Then follow all the steps that the terminal will ask you, and confirm them with `y`.

2.
If the above doesn't work for you, what I'd suggest is to re-write the entire environment. When using the `conda` terminal, just paste the below command (on Mac you may need `Command+V` or `Command+SHIFT+V`? I'm not sure). Though make sure that you're not already in the `ab_dev` environment. You can do this by typing `conda deactivate` if you see `(ab_dev)` already on the left of line you're typing in. You should see `(base)` at the start of the line you're typing at.

Use this following command:
```
conda create -n ab_dev activity-browser-dev python=3.9
```
This will ensure that the right python version is installed from the start.

When you run the command, it will ask you something like this:
```
WARNING: A conda environment already exists at 'YOURPATH'
Remove existing environment (y/[n])? 
```
Just choose `y` and follow the rest of the installation like normal.


Please let us know if this solves your issue or whether you need additional help!

**Comment by moorotter** [comment]
Hi mark-vdm,

thank you for your help! I already used the command you said to install version 3.9, then checked via python --version that it is indeed version 3.9. When I went through the process of installing the activity browser, I get the error I mentioned and if I check again which version is being used, it gives me version 3.10 even though I didn't change it. So somehow during the installation or creation of the environment of the activity browser, the python version is changed to 3.10 which I think might cause the error.

I tried your second option as well but now I am getting the following error:
CommandNotFoundError: No command 'conda activity-browser-dev'.

What should I do? Thank you!
